- Recognize PT_OPENBSD_MUTABLE with LLVM's readobj / objdump.
- correct alignment
- Implement support for PT_OPENBSD_NOBTCFI in lld(1).  This can be set using
  the -z nobtcfi option.

Index: llvm/tools/llvm-objdump/ELFDump.cpp
--- llvm/tools/llvm-objdump/ELFDump.cpp.orig
+++ llvm/tools/llvm-objdump/ELFDump.cpp
@@ -244,13 +244,19 @@ static void printProgramHeaders(const ELFFile<ELFT> &O
       outs() << "    NOTE ";
       break;
     case ELF::PT_OPENBSD_BOOTDATA:
-      outs() << "    OPENBSD_BOOTDATA ";
+      outs() << "OPENBSD_BOOTDATA ";
       break;
+    case ELF::PT_OPENBSD_MUTABLE:
+      outs() << "OPENBSD_MUTABLE ";
+      break;
+    case ELF::PT_OPENBSD_NOBTCFI:
+      outs() << "OPENBSD_NOBTCFI ";
+      break;
     case ELF::PT_OPENBSD_RANDOMIZE:
-      outs() << "    OPENBSD_RANDOMIZE ";
+      outs() << "OPENBSD_RANDOMIZE ";
       break;
     case ELF::PT_OPENBSD_WXNEEDED:
-      outs() << "    OPENBSD_WXNEEDED ";
+      outs() << "OPENBSD_WXNEEDED ";
       break;
     case ELF::PT_PHDR:
       outs() << "    PHDR ";
